modules/im_cloudbuild_workspace/cb.tf

```diff
@@ -72,13 +72,20 @@ resource "google_cloudbuild_trigger" "triggers" {
   repository_event_config {
     repository = google_cloudbuildv2_repository.repository_connection.id
     dynamic "pull_request" {
-      for_each = each.key == "preview" ? [1] : []
+      for_each = (each.key == "preview") && (local.is_gh_repo) ? [1] : []
       content {
         branch          = var.im_deployment_ref
         invert_regex    = false
         comment_control = var.pull_request_comment_control
       }
     }
+    dynamic "push" {
```

Comment:
Could we open a ticket with CB team for clarifying behavior? Does this also happen via UI?

Reply:
I created a bug to remind me to get a working example for the ticket with additional information where needed.